Bottom: ffeaa475406872d1ff74177bdce43fd9fb6ad93a
Top:    215d28234913c069c6d015ac48dbfc21e8659a04
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 13:54:08 +0100

Refresh of convert-board-objects-to-glist

---

diff --git a/src/hid/gtk/gui-pinout-preview.c b/src/hid/gtk/gui-pinout-preview.c
index 35ef8eb..46806da 100644
--- a/src/hid/gtk/gui-pinout-preview.c
+++ b/src/hid/gtk/gui-pinout-preview.c
@@ -90,7 +90,6 @@ pinout_set_data (GhidPinoutPreview * pinout, ElementType * element)
    * set all package lines/arcs to zero width
    */
   CopyElementLowLevel (NULL, &pinout->element, element, FALSE, 0, 0);
-
   PIN_LOOP (&pinout->element);
   {
     SET_FLAG (DISPLAYNAMEFLAG, pin);
